src/test/java/seedu/address/model/person/UniquePersonListTest.java

```diff
@@ -24,7 +24,7 @@ public class UniquePersonListTest {
 
     @Test
     public void contains_nullPerson_throwsNullPointerException() {
-        assertThrows(NullPointerException.class, () -> uniquePersonList.contains(null));
+        assertThrows(NullPointerException.class, () -> uniquePersonList.contains((Person) null));
```

Comment:
Actually, of this test case is to test for nullPerson, why do you need to cast null with Person? I not too sure what this means

Reply:
When the person creates an AddCommand, they use the name to find the Seller. Current methods use the Person object, so we need to overload it with an option to find with name since name is unique. However, null fits both so have to cast to Person